pkg/odo/commonflags/context/context.go

```diff
@@ -0,0 +1,49 @@
+package context
+
+import (
+	"context"
+
+	"github.com/redhat-developer/odo/pkg/odo/commonflags"
+)
+
+const (
+	OutputFlag = "outputFlag"
+)
+
+type (
+	outputKeyType struct{}
+	runOnKeyType  struct{}
+)
+
+var (
+	outputKey outputKeyType
+	runOnKey  runOnKeyType
+)
+
+// WithOutput sets the value for the output flag (-o) in ctx
+func WithOutput(ctx context.Context, val bool) context.Context {
+	return context.WithValue(ctx, outputKey, val)
+}
+
+// GetOutput gets value of output flag (-o) in ctx
+func GetOutput(ctx context.Context) bool {
```

Comment:
Same comment as above:

Would it make sense to have this function named `IsJsonOutput`? Or make it return the actual output value as a constant?
(I am thinking of cases where we might need to support other output types in the future.)